### DISCUSSION TITLE
Telegram notification for Runelite notifications

### DISCUSSION THREAD

**Discussion by Truesh00ter** [runelite/runelite#17094] [Ideas]
Simply put, I would like to receive a Telegram notification when Runelite sends out a notification and this should be easy to toggle ON and OFF. _I'm aware of the possibilities for Telegram bots and use them myself for a summary for my daily solar power generation._ I've noticed that for AFK-able skills or tasks that allow minutes of AFK-time I tend to wander about and do some chores, currently star mining is a great example where afking for minutes at a time is very possible, but doing a slayer task and getting a notification on a superiour spawn or a specific highlighted item are valid reasons on why someone would like to get a notification on their phone while RL is running. Getting a notification on my phone (via Telegram for instance) seems like a feasible way to notify someone away from their workstation. Any other free and privacy friendly way to receive notifications on a phone would be acceptable alternatives. 

The main problem is see with this is being able to explain most of the RL users how to integrate a Telegram bot with their Runelite client.

To summarize:
What: sent notifications to handheld devices when RL sends a notification to the OS it's running on.
Why: to receive notifications while being away from the workstation that is running RL.
How: possible integration with Telegram for a low effort notification on multiple handheld devices and more.

**Comment by perrylzx** [comment]
i made something like this, check out telegram notifier plugin
